unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDController.cs

```diff
@@ -1018,22 +1028,26 @@ private void FetchAndShowThirdPartyCollection(string collectionId, string collec
                 view.BlockCollectionsDropdown(false);
                 LoadUserProfile(userProfile, true);
                 view.RefreshSelectorsSize();
-            })
-            .Catch((error) =>
+            }
+            catch (Exception e)
             {
                 view.BlockCollectionsDropdown(false);
-                Debug.LogError(error);
-            });
+                Debug.LogError(e.Message);
+            }
+        }
+
+        view.BlockCollectionsDropdown(true);
+        RequestThirdPartyWearablesAsync(CancellationToken.None).Forget();
```

Comment:
Pass a proper cancellation token, please review other cases on your own (As I commented the similar one)

Reply:
Done: https://github.com/decentraland/unity-renderer/pull/4292/commits/8a52a03129307c9417ad3230813839d9b7c9ba88